core/adminserver/admin_server.go

```diff
@@ -240,11 +240,18 @@ func (s *Server) initialize(ctx context.Context, logTree, mapTree *tpb.Tree) err
 	logID := logTree.GetTreeId()
 	mapID := mapTree.GetTreeId()
 
-	logRoot, err := s.tlog.GetLatestSignedLogRoot(ctx,
-		&tpb.GetLatestSignedLogRootRequest{LogId: logID})
+	logClient, err := client.NewFromTree(s.tlog, logTree)
+	if err != nil {
+		return fmt.Errorf("could not create log client: %v", err)
```

Comment:
Some of the errors in this code are prefixed "adminserver:" and some aren't.

Reply:
They all have the `adminserver:` prefix now